src/openforms/contrib/brk/service.py

```diff
@@ -1,16 +1,26 @@
+from glom import glom
+
 from openforms.forms.models.form import Form
 from openforms.plugins.exceptions import InvalidPluginConfiguration
 
 from .checks import BRKValidatorCheck
+from .models import BRKConfig
 
 
 def check_brk_config_for_addressNL() -> str:
     live_forms = Form.objects.live()
 
-    if any(form.has_component("addressNL") for form in live_forms):
-        try:
-            BRKValidatorCheck.check_config()
-        except InvalidPluginConfiguration as e:
-            return e.args[0]
+    for form in live_forms:
+        components = form.iter_components()
+        for component in components:
+            if (
+                component["type"] == "addressNL"
+                and glom(component, "validate.plugins", default=[])
+                and BRKConfig.get_solo().service
+            ):
```

Comment:
```suggestion
            if (
                component["type"] == "addressNL"
                and (plugins := glom(component, "validate.plugins", default=[]))
                and "brk-zakelijk-gerechtigd" in plugins
                and BRKConfig.get_solo().service
            ):
```

only if this particular plugin is used do we need to perform the check, if other plugins are used that don't use the BRK, we don't want this false positive to happen again.

It would be a good idea to define that plugin ID as a constant in `src/openforms/contrib/brk/validators.py` instead of hardcoding it